### ISSUE TITLE
Error Report: Cannot read property 'value' of undefined

### ISSUE THREAD

**Issue by markrosenbaum** [urbit/bridge#1002]
## Context

I tried to log into bridge with a master ticket instead of an ETH address. 

I was not logged in yet but my ETH address is 0x5A30e92350C61960Dbb8c7D6E9F9e1f6C487280B and my planet is ~dotnet-filpem

## Error

```
TypeError: Cannot read property 'value' of undefined
    at https://bridge.urbit.org/static/js/main.280ba267.chunk.js:1:237615
    at rs (https://bridge.urbit.org/static/js/2.e1a6ea4a.chunk.js:2:3293274)
    at qs (https://bridge.urbit.org/static/js/2.e1a6ea4a.chunk.js:2:3302841)
    at Qo (https://bridge.urbit.org/static/js/2.e1a6ea4a.chunk.js:2:3346723)
    at xl (https://bridge.urbit.org/static/js/2.e1a6ea4a.chunk.js:2:3332906)
    at Sl (https://bridge.urbit.org/static/js/2.e1a6ea4a.chunk.js:2:3332834)
    at Il (https://bridge.urbit.org/static/js/2.e1a6ea4a.chunk.js:2:3332697)
    at hl (https://bridge.urbit.org/static/js/2.e1a6ea4a.chunk.js:2:3329663)
    at https://bridge.urbit.org/static/js/2.e1a6ea4a.chunk.js:2:3279042
    at t.unstable_runWithPriority (https://bridge.urbit.org/static/js/2.e1a6ea4a.chunk.js:2:3356461)
```

In Bridge version 3.0.0.

**Comment by markrosenbaum** [comment]
This issue occurred separately from issue #1001

**Comment by tomholford** [comment]
Thanks for the report, unfortunately I am not able to reproduce this issue locally.

Does the wallet in question hold many Urbit assets?

**Comment by markrosenbaum** [comment]
This seems to have been fixed, I tested this along with issues #1001

**Comment by lukestiles** [comment]
Closing as fixed/cannot reproduce.
